File: BECore/app/model/ProjectExperiment_AttachDetail.js
﻿/// <summary>
/// Model EXTJS ProjectExperiment_AttachDetail.
/// </summary>
Ext.define('QLDT.model.ProjectExperiment_AttachDetail', {
    extend: 'Ext.data.Model',
    idProperty: 'ProjectExperiment_AttachDetailID',
    idType: 'Guid',
    fields: [
		{ name: "ID", type: "string" },
        { name: "Description", type: "string" },
		{ name: "FileName", type: "string" },
		{ name: "FileType", type: "string" },
		{ name: "FileSize", type: "string" },
		{ name: "SortOrder", type: "int" },
		{ name: "CreatedDate", type: "date" },
		{ name: "ModifiedDate", type: "date" },
		{ name: "IPAddress", type: "string" },
		{ name: "ModifiedBy", type: "string" },
		{ name: "CreatedBy", type: "string" },
        { name: 'FileResourceID', type: "string" },
        { name: 'IsTemp',type:'boolean' }
    ]
});

File: MT.Model/Business/ProjectExperiment_AttachDetail.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace MT.Model
{
    #region ProjectExperiment_AttachDetail:EntityBase
    /// <summary>
    /// This object represents the properties and methods of a ProjectExperiment_AttachDetail:EntityBase.
    /// </summary>
    public class ProjectExperiment_AttachDetail : EntityBase
    {
        [Key]
        public Guid ProjectExperiment_AttachDetailID { get; set; }
        public Guid ID { get; set; }
        public string Description{ get; set; }
        public string FileName { get; set; }
        public string FileType { get; set; }
        public double FileSize { get; set; }
        public int SortOrder { get; set; }

        public Guid? FileResourceID { get; set; }
    }
    #endregion
}

